client/me/help/help-contact/index.jsx

```diff
@@ -211,16 +210,14 @@ module.exports = React.createClass( {
 	},
 
 	onOperatorsAway: function() {
-		const { isOlarkReady, isUserEligible, details } = this.state.olark;
-		const showChatVariation = isUserEligible && details.isOperatorAvailable;
-		const showKayakoVariation = ! showChatVariation && ( details.isConversing || isUserEligible );
-		const showForumsVariation = ! ( showChatVariation || showKayakoVariation );
+		const { isUserEligible, details } = this.state.olark;
 
 		if ( ! details.isConversing ) {
 			analytics.tracks.recordEvent( 'calypso_help_offline_form_display', {
-				form_type: showKayakoVariation ? 'kayako' : 'forum'
+				form_type: isUserEligible ? 'kayako' : 'forum'
```

Comment:
I felt I should show my work here just so someone else could try and spot a mistake. Here's how I reduced it

```js

showKayakoVariation = ! ( isUserEligible && details.isOperatorAvailable ) && ( details.isConversing || isUserEligible ); // substitute showChatVariation for its actual value
showKayakoVariation = ( ! isUserEligible || ! details.isOperatorAvailable ) && ( details.isConversing || isUserEligible ); // convert ! ( isUserEligible && details.isOperatorAvailable ) to ( ! isUserEligible || ! details.isOperatorAvailable )
showKayakoVariation = ( ! isUserEligible || ! false ) && ( details.isConversing || isUserEligible ); // We already know operators aren't available so lets replace details.isOperatorAvailable with false
showKayakoVariation = ( ! isUserEligible || true ) && ( details.isConversing || isUserEligible ); //simplify
showKayakoVariation = true && ( details.isConversing || isUserEligible ); //simplify
showKayakoVariation = details.isConversing || isUserEligible; //simplify
showKayakoVariation = false || isUserEligible; // Our case below indicates that details.isConversing should be false so lets fill it in
showKayakoVariation = isUserEligible; // simplify

```